modules/querying/pages/query-operations.adoc

```diff
@@ -981,7 +1077,10 @@ Additionally, the `ls` GSQL command lists all created queries and identifies whi
 dropQuery := DROP QUERY ( "*" | ALL | queryName ["," queryName]* )
 ----
 
-To drop a query, run `DROP QUERY query_name` __. The query will be uninstalled (if it has been installed) and removed from the dictionary.  The GSQL language will refuse to drop an installed query if another query is installed which calls query Q.  That is, all calling queries must be dropped before or at the same time that their called subqueries are dropped.
+To drop a query, run `DROP QUERY query_name`.
+The query will be uninstalled (if it has been installed) and removed from the dictionary.  The GSQL language will refuse to drop an installed query if another query is installed which calls query Q.  That is, all calling queries must be dropped before or at the same time that their called subqueries are dropped.
```

Comment:
what's `query Q`?

Reply:
I remember "we can drop a subquery, and the main query (no matter installed or not) will be marked as disabled", @xun-gao-tigergraph please test it out.

Reply:
Thanks for the example!